SimpleAkkaDotNetCluster/SimpleClusterCaller/ClusterClientActor.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Immutable;
+using Akka.Actor;
+using Akka.Cluster.Tools.Client;
+using Akka.Event;
+using SimpleClusterLib;
+
+namespace SimpleClusterCaller;
+
+public class ClusterClientActor: ReceiveActor
```

Comment:
This actor is required because of the quirks of how external `Ask<T>()` and `ClusterClientReceptionist` works.
- When you do an `Ask<T>()` from outside an actor system, a temporary actor is created to wait for the response. The `ClusterClientActor` work as a translator that stores that temporary actor reference inside an envelope so that it can be addressed later.
- The `ClusterClientReceptionist` have a quirk that when it dispatches messages to the registered actors, it will use itself as the Sender. When the registered actor received a message, it will see all messages as being sent by the `ClusterClientReceptionist`. To get around this problem, we're wrapping the message payload in an envelope that contains a reference to the original sending actor, in this case the temporary actor created by the `Ask<T>()`.